packages/e2e.sandbox.views.todo-widget/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@osdk/e2e.sandbox.views.todo-widget",
```

Comment:
note i haven't actually ran this yet, i mostly made it to make sure all the types are right. I will go after a proper template (+ vite plugin) in a follow up PR